addons/website_livechat/static/src/js/website_livechat.js

```diff
@@ -0,0 +1,34 @@
+odoo.define('website_livechat.history', function (require) {
+    var local_storage = require('web.local_storage');
+    var session = require('web.session');
+    var bus = require('bus.bus').bus;
+    var website = require('website.website');
+    var im_livechat = require('im_livechat.im_livechat');
+
+    var page = window.location.href.replace(/^.*\/\/[^\/]+/, ''),
+        page_history = local_storage.getItem('visited_page_history');
+    var url_history = JSON.parse(page_history) || [];
+    if(!_.contains(url_history, page)){
+        url_history.push(page);
+        local_storage.setItem('visited_page_history', JSON.stringify(url_history));
```

Comment:
`localstorage` is not unlimited. Keep only the last 15 visited urls, in their order.
And adapt the command message accordingly.